src/screens/ProfileScreen/index.js

```diff
@@ -91,29 +97,45 @@ const Header = (props) => {
     setTabIndexToSigned,
     setTabIndexToAnonymous
   } = props;
-
+  const navigator = useNavigation();
+  const [feedsContext, dispatch] = React.useContext(Context).feeds;
+
+  const {feeds, timer, viewPostTimeIndex} = feedsContext;
+  // eslint-disable-next-line no-underscore-dangle
+  const handleOnAddPostButtonClicked = () => {
+    const currentTime = new Date().getTime();
+    const id = feeds && feeds[viewPostTimeIndex]?.id;
+    if (id) viewTimePost(id, currentTime - timer.getTime(), SOURCE_FEED_TAB);
+    navigator.navigate(NavigationConstants.CREATE_POST_SCREEN);
+    setTimer(new Date(), dispatch);
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for handling post creation time has been updated. This could potentially affect how posts are displayed or sorted. Ensure that this change is thoroughly tested.
```diff
+ const navigator = useNavigation();
+ const [feedsContext, dispatch] = React.useContext(Context).feeds;
+ const {feeds, timer, viewPostTimeIndex} = feedsContext;
+ const handleOnAddPostButtonClicked = () => {
+   const currentTime = new Date().getTime();
+   const id = feeds && feeds[viewPostTimeIndex]?.id;
+   if (id) viewTimePost(id, currentTime - timer.getTime(), SOURCE_FEED_TAB);
+   navigator.navigate(NavigationConstants.CREATE_POST_SCREEN);
+   setTimer(new Date(), dispatch);
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->